setup_default_backend.sh

```diff
@@ -51,6 +51,8 @@ function install_backend_python_packages {
     pip install -r "requirements-build.txt" && loggood "Successfully installed pip and setuptools" || logerror "Failed to install pip and setuptools"
     log "Installing dependencies from requirements.txt..."
     pip install -r "requirements.txt" && loggood "Successfully installed the dependencies in requirements.txt" || logerror "Failed to install dependencies in requirements.txt"
+    log "Installing all previews dependencies from requirements-full-preview-generator.txt"
+    pip install -r "requirements-full-preview-generator.txt"  && loggood "Successfully installed the dependencies in requirements-full-preview-generator.txt" || logerror "Failed to install dependencies in requirements-full-preview-generator.txt"
```

Comment:
Should we actually keep installing all these dependencies by default in setup_default_backend? The docker image does not use this script, setup_default_backend is probably mostly used by developer and we rarely use these builders.

This is logical to keep the current behavior, but I'm still asking just in case

Reply:
For first step i suggest to keep current behavior.
I think, we should have some kind of tweaking support with env_var or command option a bit later.
This PR is more about upgrading preview_generator and less about taking profit of the new behavior.

If you have nice suggestion about how to make it parametrizable and why (don't clearly know how the new change affect the 20.04 support for example), you can add it to the new issue expect to create.